src/utils/QuerySummaryUtils.ts

```diff
@@ -0,0 +1,123 @@
+import * as Globalize from 'globalize';
+import { IQuerySuccessEventArgs } from '../events/QueryEvents';
+import { $$ } from './Dom';
+import { l } from '../strings/Strings';
+import { IQuery } from '../rest/Query';
+import { Component } from '../ui/Base/Component';
+import { any } from 'underscore';
+import { get } from '../ui/Base/RegisteredNamedMethods';
+import { resultListId } from '../ui/ResultList/ResultListConstants';
+import { IResultListOptions } from '../ui/ResultList/ResultList';
```

Comment:
This seems like a bad import, no ? Should be imported from `IResultListOptions.ts`

Reply:
Moved the interface into it's own file just in case. Importing from the ResultList likely works because an interface is a Typescript construct, not rooted in javascript.